app/views/events/timeline.html.haml

```diff
@@ -29,27 +29,23 @@
             %p
               = event.description
             %p.datetime
-              %span
-                Starting at:
-              %i.fa.fa-clock-o
-              = event.start_at.strftime("%H:%M")
-              &nbsp;
-              %i.fa.fa-calendar
-              = event.start_at.strftime("%d.%m")
+              Starting at:&ensp;
+              %span.time
+                = event.start_at.to_s(:time)
+              %span.date
+                = event.start_at.strftime("%d.%m")
             %p.datetime
-              %span
-                Ending at:
-              %i.fa.fa-clock-o
-              = event.end_at.strftime("%H:%M")
-              &nbsp;
-              %i.fa.fa-calendar
-              = event.end_at.strftime("%d.%m")
+              Ending at:&emsp;
+              %span.time
+                = event.end_at.to_s(:time)
+              %span.date
+                = event.end_at.strftime("%d.%m")
           .card-action
             - if can?(:edit, event) && event.persisted?
               %a{ href: url_for([:edit, event])} Edit event
             .avatars
               - event.attendances.each_with_index do |a, i|
-                = image_tag a.guest.avatar_url, class: ['circle', "attendance-state-#{a.state}"], style: "right: #{i*40}px"
+                = image_tag a.guest.avatar_url, class: 'circle', style: "right: #{i*40}px"
```

Comment:
I think this is a merge error, the `attendance-state-` class should still be present.